1. line 9 prints "values added: 20"
2. line 13 prints "final result: 20"
3. line 9 prints "values added: 20"
4. line 13 returns an error because the let keyword makes it so result can only be accessed in the block it was defined.
5. line 9 returns an error but because it never gets to that line since you are trying to modify avariable with the const keyword
6. line 13 returns an error but because it never gets to that line since you are trying to modify avariable with the const keyword